### ISSUE TITLE
Format number choice

### ISSUE THREAD

**Issue by pjam** [symfony2admingenerator/GeneratorBundle#148]
In list view, when only 1 object is created, it reads "Showing all 1 results", where it should be "Showing all 1 result" or "Showing 1 result".

**Comment by loostro** [comment]
This can be changed by editing the `nbresults.php.twig` template [here](https://github.com/symfony2admingenerator/GeneratorBundle/blob/master/Resources/templates/CommonAdmin/ListTemplate/nbresults.php.twig#L12) adding another if 

(if results > 1  ---> current behaviour, if results == 1 --> new message)

**Comment by sescandell** [comment]
The right way to do is to use a `trans_choice` in place of `trans`

**Comment by pjam** [comment]
Yeah, that was my first thought, but looking at the code I see I would have to create the echo_transchoice method and for me its not worth the trouble, I'll go for the condition loostro suggested.
Anyway the issue is here for any contributor to take it and solve it.

**Comment by pjam** [comment]
Is it possible to override the nbresults template without changing the vendor file? I've been trying to do it without success...

**Comment by bobvandevijver** [comment]
@pjam Yes, but you will have to follow this cookbook item: [Extending generator templates](https://github.com/symfony2admingenerator/AdmingeneratorGeneratorBundle/blob/master/Resources/doc/cookbook/extending-generator-templates.md).

**Comment by pjam** [comment]
Thanks @bobvandevijver exactly what I needed!
